english/cycling-backwards/cycling-backwards.md

```diff
@@ -3,7 +3,16 @@ Unlike brown bears, who can only play balalaika, and only in Russia.
 
 ...but what about other animals that can cycle forwards.  [What do they do?](../cycling-forwards/cycling-forwards.md)
 
-[That's got to be horse...note to self.](../marshmallow.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.